### ISSUE TITLE
How are we going to handle versioning?  I suggest Semantic Versioning 2.0.0

### ISSUE THREAD

**Issue by mrobustelli** [tsdservices/Gravity#6]
http://semver.org/spec/v2.0.0.html

**Comment by rantowork** [comment]
I think this is an acceptable strategy for an open source solution like this.  In the same vain, it might make sense to also consider semantic git for consistency in commits.

**Comment by mrobustelli** [comment]
Is this what you are talking about? https://seesparkbox.com/foundry/semantic_commit_messages

**Comment by rantowork** [comment]
Yep, for the most part.  We use this strategy on the product team here at Milyli and have had a lot of success with it.  Strict adherence to git flow and semantic git has been a really big boost to what we do.  Here's a quick snapshot of our Delegate commits using this:

![image](https://user-images.githubusercontent.com/2838721/32120149-29211fc0-bb1d-11e7-885b-647dc7647960.png)

**Comment by Arithmomaniac** [comment]
Notes from the 2017-12-13 call:
- Want to bump minor release versions (and below) only, until product is stable
- Switch to SemVer at some point in the future

**Comment by Arithmomaniac** [comment]
2017-12-13 meeting: this will be deferred until the API is stable.
